client/src/app/site/pages/organization/pages/accounts/services/common/account-filter.service.ts

```diff
@@ -164,6 +164,14 @@ export class AccountFilterService extends BaseFilterListService<ViewUser> {
                         label: this.translate.instant(`Same email`)
                     }
                 ]
+            },
+            {
+                property: `hasMemberNumber`,
+                label: this.translate.instant(`Member number`),
```

Comment:
Please remove `this.translate.instant`
adding this creates the following bug: https://github.com/OpenSlides/openslides-client/issues/3580

I already created an issue for the preexisting filter terms (#3684

Reply:
Done, removed the translate.instant calls from the member number filter.